packages/components/src/DataList/DataList.tsx

```diff
@@ -12,13 +12,18 @@ import {
   generateListItemElements,
   getCompoundComponent,
 } from "./DataList.utils";
+import { Text } from "../Text";
+import { Glimmer } from "../Glimmer";
+import { Heading } from "../Heading";
 
 export function DataList<T extends DataListObject>({
   data,
   headers,
   loading = false,
   filterApplied = false,
   children,
+  title,
+  totalCount = 0,
```

Comment:
if you implement the data list without specifying the totalCount

```tsx
<DataList data={...} headers={...}>
  ...
</DataList>
```

It'll always say 0 results